app/controllers/checkout_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CheckoutController < ApplicationController
```

Comment:
The controller name should be plural, the filename should be checkouts_controller.rb, and the class name should be CheckoutsController.